programs/castle-vault/src/instructions/consolidate_refresh.rs

```diff
@@ -88,43 +88,7 @@ pub fn handler<'info>(ctx: Context<'_, '_, '_, 'info, ConsolidateRefresh<'info>>
         let allocation: SlotTrackedValue = ctx.accounts.vault.actual_allocations[p];
```

Comment:
In the vault value calculation we also need to take into account the value of the port rewards.

Reply:
I skipped for now because port reward is a different token, and I haven't looked at how to find its exchange rate. Probably the rates provided by the `StakingPool` struct, but I'm not sure.

Also we have to claim and sell reward tokens before the reward value can be used by rebalancing, but these are not implemented yet. Having reward token value added here now wouldn't be correct.

IMO it's better to not add the reward token value here, but instead periodically claim and sell the reward token and add the revenue from the sales directly to the vault reserve.